app/routes/todos.tsx

```diff
@@ -32,6 +35,17 @@ export async function action({ request }: Route.ActionArgs) {
       }
       break;
     }
+    case "move": {
+      const id = formData.get("id") as any;
+      const target = formData.get("target") as unknown as string;
+      if (typeof id === "string" && typeof target === "string") {
+        moveTodo(id, target as any);
```

Comment:
suggestion (types): validate issue status on every action for compatibility with new and old status and types
```suggestion
        // Validate target with isValidStatus
        if (isValidStatus(target)) {
          moveTodo(id, target);
        } else {
          return { error: "Invalid status" };
        }
```